Library/Homebrew/cmd/tap.rb

```diff
@@ -42,59 +41,17 @@ def install_tap user, repo
     true
   end
 
-  def link_tap_formula(paths, warn_about_conflicts=true)
-    ignores = (HOMEBREW_LIBRARY/"Formula/.gitignore").read.split rescue []
-    tapped = 0
+  # migrate from symlink based tap structure to directories based structure
+  def migrate_taps(options={:autoskip => true})
```

Comment:
This isn't really the best way to add an option the defaults to true... if someone calls the method with a hash argument but omits the autoskip option, it will be falsy instead. I think this is more idiomatic:

```rb
def migrate_taps(options={})
  autoskip = options.fetch(:autoskip, true)
```